pkg/cmd/types.go

```diff
@@ -59,3 +65,44 @@ func (i *ImageInspectorOptions) Validate() error {
 	}
 	return nil
 }
+
+// OpenSCAPStatus is the status of openscap scan
+type OpenSCAPStatus string
+
+const (
+	StatusNotRequested OpenSCAPStatus = "NotRequested"
+	StatusSuccess      OpenSCAPStatus = "Success"
+	StatusError        OpenSCAPStatus = "Error"
+)
+
+type OpenSCAPMetadata struct {
+	Status           OpenSCAPStatus // Status of the OpenSCAP scan report
```

Comment:
I meant for the ```SetError``` function to be just a convenience function since I noticed that I used those three lines multiple times. I think this structure is mostly just a group of data and less a real type so adding accessors seems a bit overkill.